adding new text Job and I discussed

ksriram25 opened the pull request:
Added text discussed with Job:

> * Specify/clarify that more specific prefixes of an included prefix 
> are considered Invalid if announced with the PrefixList AS as the 
> origin AS. Yes? I think we had agreed to this.

JS: Yes, I believe so too.

> * Define Validated PrefixList Payload

JS: Yes

> * Add a simple brief section on the problems solved by PrefixList 
> (complementing the ROA solution)

JS: Yes

JS: Please submit a PR for the above 3 points.

ksriram25 committed:
adding new text Job and I discussed

job commented:
We should call it a VSP, not a VPP. `VPP` is already widely used in networking as an abbrevation. Can you change and push an additional commit into this branch?

ksriram25 committed:
changed VPP to VSP

ksriram25 commented:
@job 
> We should call it a VSP, not a VPP. `VPP` is already widely used in networking as an abbrevation. Can you change and push an additional commit into this branch?

Done.

job commented:
Hi @ksriram25 - please place exactly one sentence on each line. This makes review much easier, and does not alter the final output when rendered to txt or HTML.

I see that some sentences are spread across multiple lines, and that some lines contain multiple sentences.

You should be able to make the adjustments and commit and push the commit in order for  it to show up in this pull-request, no need to rebase or close this PR.

ksriram25 committed:
once sentence per line

ksriram25 commented:
@job 
> You should be able to make the adjustments and commit and push the commit in order for it to show up in this pull-request, no need to rebase or close this PR.

Yes. All done.

job commented on draft-ietf-sidrops-rpki-prefixlist.xml:
```diff
@@ -193,21 +216,86 @@
+        Normally there would be a single valid Signed Prefix List object for a given asID (see <xref target="operations"/>).
+        However, multiple valid Signed Prefix List objects which contain the same asID could exist.
+        In such cases, the union of the resulting address prefix members forms the complete set of members.
+        The complete set (resulting from a single or multiple PrefixLists) is locally stored by a relying party, and such an object is called the Validated Signed-PrefixList Payload (VSP) for the asID in consideration.
```
please remove the dash between Signed and PrefixList (aka SignedPrefixList)

ksriram25 replied:
Done

job commented on draft-ietf-sidrops-rpki-prefixlist.xml:
```diff
@@ -193,21 +216,86 @@
+        However, multiple valid Signed Prefix List objects which contain the same asID could exist.
+        In such cases, the union of the resulting address prefix members forms the complete set of members.
+        The complete set (resulting from a single or multiple PrefixLists) is locally stored by a relying party, and such an object is called the Validated Signed-PrefixList Payload (VSP) for the asID in consideration.
+        (Note: The VSP is akin to the Validated ROA Payload (VRP) for Route Origin Authorization (ROA) <xref target="RFC6811"/>.)
```
I am not sure this note is helpful, as in - yes Validated Payloads are the result of the validation process, but in what way exactly are they akin? That's not explained here and probably shouldn't be explained here

ksriram25 replied:
Done. Removed the sentence starting with (Note: ....

ksriram25 committed:
a couple of edits -- remove dash etc.

s/Validated Signed-PrefixList Payload (VSP)/Validated SignedPrefixList Payload (VSP)/

Removed the sentence starting with:
(Note: The VSP is akin to the Validated ROA Payload (VRP)....

ksriram25 commented:
@job 
I have used 'Signed PrefixList', 'PrefixList verification' etc.
That is, I preferred to use PrefixList (no space) instead of Prefix List (with space).
The document uses Prefix List (with space) in other places. 
OK? Or we need to get consistence?
I tend to prefer PrefixList or Signed PrefixList because PrefixList is an object and naming it using one word seems better to me rather than Prefix List (with space).
It is your call. Please let me know. I'll be happy to s/PrefixList/Prefix List/ in the parts of the text I contributed.

job commented on draft-ietf-sidrops-rpki-prefixlist.xml:
```diff
@@ -193,21 +216,85 @@
+        Thus, AS A protects itself from this form of AS abuse.
+        -->
+        <li>
+        <strong>AS Abuse Together with a Bogus ROA:</strong>
```
The terminology "Bogus ROA" is not optimal, because the ROA is cryptographically valid, and the IP Prefix Resource holder is free to list any ASN they want in the origin field.

ksriram25 replied:
How about 'malicious ROA' or 'a ROA created with malicious intent'?
Another word that comes to mind is 'deceptive ROA'.

job replied:
“Non-reciprocal ROA” - because we can only guess about intent, and it might not even be malicious, but with SPL the RPs can conclude the Roa is not reciprocated. Thoughts?

ksriram25 replied:
That seems like a more appropriate name. 

Proposed new text:

2.  *AS Abuse Together with a Non-Reciprocal ROA:*  A non-reciprocal ROA is one which attests the origination of a prefix from an AS but the AS has not included the specific prefix in its Signed Prefix List. Consider the scenario where AS A creates a Signed Prefix List. In this case, AS A is protected if an offending AS X inserts AS A as the origin AS and announces a prefix for which AS X holds a certificate and has signed a non-reciprocal ROA showing AS A as the origin.

ksriram25 replied:
@job 
Rethinking... we are describing an "attack type" here in which the offending AS X is engaged in a forged-origin prefix hijack (intentionally). So, it should be OK to use the term 'malicious ROA' which readers can readily understand. Anyway, will wait to hear your pick from the available choices.

ksriram25 commented:
About to create PR#6 which is re-based on the current Main and includes all changes from this PR (#4) and additional changes requested by Job recently.
